packages/preview/kinase/README.md

```diff
@@ -0,0 +1,27 @@
+# Kinave
```

Comment:
There should only be a sub-folder per version in this directory, so I think you should delete this file.